dip-0027.md

```diff
@@ -0,0 +1,163 @@
+
+## Asset Unlocking
+
+Withdrawing locked credit pool funds back to Dash is done by creating a Core chain unlock transaction signed by a Platform validator quorum (e.g., LLMQ_100_67 for mainnet). This transaction will release Dash from the credit pool while reducing the credit pool balance by an equivalent amount.
+
+### Asset Unlock Transaction
+
+The Asset Unlock transaction (AssetUnlockTx) is a DIP-2-based special transaction that enables the withdrawal of credit pool funds as Dash. This special transaction is type 9.
+
+Note: Asset Unlock transactions have no inputs. Their uniqueness is instead derived from the index they contain, which must be unique.
+
+The format of the Asset Unlock special transaction payload:
+
+| **Field** | **Type** | **Size** | **Description** |
+| - | - | - | - |
+| version | uint8_t | 1 | Asset Unlock version number. Currently set to 1 |
+| index | uint64 | 8 | The index of the transaction |
+| fee | uint32 | 4 | The miner's fee in duffs |
+| signHeight | uint32 | 4 | The height of the Core chain known by Platform at the moment of the Asset Unlock signing (Core ChainLock height) |
+| quorumHash | uint256   | 32 | Quorum hash. Set by the current validator. Must refer to an active quorum at `signHeight` |
+| quorumSig | uint768   | 96 | Threshold signature recovered during Platform Consensus. Must be a signature for the full transaction, not just the payload. |
+
+#### Signing Asset Unlock Transactions
+
+An active Platform validator quorum quorum must sign Asset Unlock transactions per the process described in [DIP-7](./dip-0007.md). The signing session’s requestID is calculated as follows:
+
+    requestID = SHA256(SHA256("plwdtx", index))
+
+For reference, the following table shows some example index values and their request IDs:
+
+| **Index** | **Request ID** |
+|-----------|----------------|
+| 101       | fcc76a643c5c668244fdcef09833955d6f4b803fa6c459f7732983c2332389fd |
+| 123456789 | ebd9d75ad72184bec8e8f25a499eb5386ba564eb7ce70a4a3b9db652599d6d72 |
+
+The SignID is defined as _SignID = SHA256(quorumHash, requestID, sigHash)_ where sigHash is the hash of the entire transaction with the quorumSig field set to zeros. The transaction must be hashed as described in [DIP-2](./dip-0002.md#serialization-hashing-and-signing).
+
+### Client verification
+
+All clients must verify the quorumSig field before accepting the transaction. If the quorum is recent and the transaction is not yet mined, light clients should attempt to verify the quorumSig by retrieving the quorum public key as described in DIP-4 and verifying the signature. If the quorum is not recent and the transaction is not mined, the transaction should be ignored until it is mined to prevent attacks on the light client.
```

Comment:
what does `recent` mean?

Reply:
Resolved in https://github.com/dashpay/dips/pull/152#discussion_r1771788437